src/services/getItemsFromFirebase.tsx

```diff
@@ -1,8 +1,9 @@
 import { db } from '../firebase';
 import { collection, getDocs } from 'firebase/firestore';
+import { CategoryTypes } from '../types/CategoryTypes';
 
 export const getItemsFromFirebase = async (
-  category: string,
+  category: CategoryTypes,
```

Comment:
I teraz jak wywołujesz tą funkcję to przekazujezs string, czy np. CategoryTypes.WEAPONS ? :) to wazne, bo pewnie TS będzie krzyczał jesli przekażesz zwykły string